marklogic-data-hub-central/ui/src/components/common/confirm-yes-no/confirm-yes-no.tsx

```diff
@@ -14,9 +15,10 @@ type Props = {
 
 const ConfirmYesNo: React.FC<Props> = (props) => {
```

Comment:
i dont think that's within scope, because this isnt the only component that needs destructuring . We should definetly create a task that tackles that OR change the scope and SP to a higher number. Also remember that the task that you have is a clone, and therefore should apply destructuring in every component you come across too.